basculehttp/bearerTokenFactory.go

```diff
@@ -26,7 +26,7 @@ import (
 	"github.com/golang-jwt/jwt"
 	"github.com/xmidt-org/arrange"
 	"github.com/xmidt-org/bascule"
-	"github.com/xmidt-org/bascule/key"
+	"github.com/xmidt-org/clortho"
 	"go.uber.org/fx"
```

Comment:
We're not using `bascule` anymore after this, right? If so, we should probably rename files, types and the dirs that refer to the name `bascule`.